packages/react/src/components/HotspotEditorModal/HotspotEditorDataSourceTab/HotspotEditorDataSourceTab.test.jsx

```diff
@@ -202,7 +198,6 @@ describe('HotspotEditorDataSourceTab', () => {
     render(
       <HotspotEditorDataSourceTab
         hotspot={cardConfigWithPresets.content.hotspots[0]}
-        thresholds={cardConfigWithPresets.thresholds || []}
```

Comment:
I think these were removed b/c thresholds are passed as part of the hotspots not their own props, right?

Reply:
Yes, they weren't being used here. I'm also gonna create an issue for removing the thresholds array of the cardConfig prop passed to the HotspotEditorModal since according to https://github.com/carbon-design-system/carbon-addons-iot-react/pull/1949 thresholds are never handled globally and all that code (that existed in the first release) has been removed.